### ISSUE TITLE
Training Loss 

### ISSUE THREAD

**Issue by hyperparameters** [Zhongdao/Towards-Realtime-MOT#203]
@Zhongdao Appreciate your work :+1: ,
can you provide training loss logs for MOT datasets for reference?
Thank you in advance.
